src/sinks/azure_common/sink.rs

```diff
@@ -48,11 +48,20 @@ where
 
         input
             .batched_partitioned(partitioner, || settings.as_byte_size_config())
+            .filter_map(|result| async move {
+                match result {
+                    Ok((key, batch)) => Some((key, batch)),
+                    Err(error) => {
+                        emit!(SinkRequestBuildError { error });
+                        None
+                    }
+                }
+            })
             .filter_map(|(key, batch)| async move {
                 // We don't need to emit an error here if the event is dropped since this will occur if the template
                 // couldn't be rendered during the partitioning. A `TemplateRenderingError` is already emitted when
                 // that occurs.
```

Comment:
Need to double check this. I think we're emitting `TemplateRenderingError` twice now. Once from inside the partitioner and once from `SinkRequestBuildError`. Again, this occurs in many places in this PR